plugins/outputs/http/README.md

```diff
@@ -0,0 +1,34 @@
+# HTTP Output Plugin
+
+This plugin writes to a HTTP Server using the `POST Method`.
+
+Data collected from telegraf is sent in the Request Body.
+
+### Configuration:
+
+```toml
+# Send telegraf metrics to HTTP Server(s)
+[[outputs.http]]
+  ## It requires a url name.
+  ## Will be transmitted telegraf metrics to the HTTP Server using the below URL.
+  ## Note that not support the HTTPS.
```

Comment:
I will add support for the regular HTTPS options once this is merged, but this code will support connecting to a HTTPS url as it is.

Reply:
Do you want to do additional work on HTTPS after you first merge it? I have not worked on this part.